packages/DataTable/src/DataTable.tsx

```diff
@@ -143,29 +157,45 @@ const DataTable: React.FC<DataTableProps> & DataTableComposition = ({
     );
   }
 
-  return (
-    <ThemeContext.Provider value={{ borderType, isHeaderSticky, hasZebraStripes }}>
+  function renderTable(maxHeight: number) {
+    return (
       <sc.Table
+        ref={tableRef}
         aria-label={a11yText}
         aria-rowcount={data.length}
         data-pka-anchor="dataTable"
         width={width}
-        height={height}
+        height={getTableHeight(maxHeight)}
```

Comment:
Can we just use `height` instead of `getTableHeight (maxHeight)`  when there's no `extractedResizeContainer ` ?